ynput/ayon-core
# Scene manager: Show statuses

## Changelog Description
Added statuses to scene inventory manager.

## Additional info
The code of scene inventory tool was not much clear so it was "easier" to refactor some parts for easier understanding where and how data are comming in. The refactor is done only in bounds of this feature. Also discovered few bugs that probably caused issues even in current develop, if you hit a bug please make sure it is bug related to this PR.

## Screenshot
![image](https://github.com/ynput/ayon-core/assets/43494761/7f26c067-075c-417b-95fb-c10edc2d3f25)
![image](https://github.com/ynput/ayon-core/assets/43494761/1001acc4-7eca-4c4d-b5d2-5ef26ef9fc2f)


## Testing notes:
1. Launch host of your choice.
2. Load/reference some representations in.
3. Scene inventory should show them.
4. You should be able to change their version, update and remove them, switch to hero and back, switch to different representation.
5. Status should show current status of version of loaded representation.
